config.py

```diff
@@ -145,6 +152,24 @@ class Configuration(object):
             "key": MINIMUM_FEATURED_QUALITY,
             "label": _("Minimum quality for books that show up in 'featured' lanes"),
         },
+    ] + [
+        { "key": ENABLED_FACETS_KEY_PREFIX + group,
+          "label": _("Enabled %(group)s facets", group=display_name),
+          "type": "list",
+          "options": [
+              { "key": facet, "label": FacetConstants.FACET_DISPLAY_TITLES.get(facet) }
+              for facet in FacetConstants.FACETS_BY_GROUP.get(group)
+          ],
+        } for group, display_name in FacetConstants.GROUP_DISPLAY_TITLES.iteritems()
```

Comment:
In the admin interface, this shows up with a heading for each group and a checkbox for each facet in the group.